include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -375,8 +376,17 @@ MultigridPreconditionerBase<dim, Number>::do_initialize_dof_handler_and_constrai
   {
     // create coarse grid triangulations only once
     if(data.involves_h_transfer() && coarse_grid_triangulations.empty())
+    {
+      AssertThrow(tria->n_global_levels() == 1 ||
+                    dynamic_cast<parallel::fullydistributed::Triangulation<dim> const *>(tria) ==
+                      nullptr,
+                  ExcMessage(
+                    "A parallel::fullydistributed::Triangulation cannot be refined if the option "
```

Comment:
Just a minor comment here regarding the message: 'To me, this sounds like the aim of this function would be to refine the `p:f:T`?

I would therefore formulate it from a multigrid perspective: "h-transfer is currently not supported for the option use_global_coarsening in combination withr a p:f:T that contains refinements. Either use a p:f:T without refinements, a p:d:T, or a MultigridType without h-transfer." Is this statement correct?

Reply:
Done.